indexing-service/src/main/java/org/apache/druid/indexing/overlord/WorkerTaskRunner.java

```diff
@@ -48,8 +48,11 @@ enum ActionType
 
   /**
    * Check which workers can be marked as lazy
+   *
+   * @param isLazyWorker   predicate that checks if a worker is lazy
+   * @param maxLazyWorkers maximum number of lazy workers to return
    */
-  Collection<Worker> markWorkersLazy(Predicate<ImmutableWorkerInfo> isLazyWorker, int maxWorkers);
+  Collection<Worker> markWorkersLazy(Predicate<ImmutableWorkerInfo> isLazyWorker, int maxLazyWorkers);
```

Comment:
Nit: This method name is probably not accurate anymore, because it does not mark workers as lazy. It gets the list of workers which are lazy and are safe to terminate. So maybe `getLazyWorkersToTerminate`?

```suggestion
  Collection<Worker> getLazyWorkersToTerminate(Predicate<ImmutableWorkerInfo> isLazyWorker, int maxToTerminate);
```

Reply:
I agree the name is funny, but I didn't want to rename it since it's an interface method.